utils/src/main/java/io/neow3j/utils/Numeric.java

```diff
@@ -79,8 +79,8 @@ public static boolean containsHexPrefix(String input) {
                 && input.charAt(0) == '0' && input.charAt(1) == 'x';
     }
 
-    public static BigDecimal toBigDecimal(byte[] value) {
-        return new BigDecimal(toBigInt(value));
+    public static BigDecimal fromFixed8ToBigDecimal(byte[] value) {
```

Comment:
Cool that you found that. :-) Maybe it was even affecting other functionalities.

Reply:
Alright. Please, add to your ToDo list to check that out soon (i.e., similar methods with the same problem).